src/ui/win32/AssetEditorDialog.cpp

```diff
@@ -138,6 +138,14 @@ class ValueColumnBinding : public ra::ui::win32::bindings::GridTextColumnBinding
 
     void SetMaximum(unsigned int nValue) noexcept { m_nMaximum = nValue; }
 
+    std::wstring GetText(const ra::ui::ViewModelCollectionBase& vmItems, gsl::index nIndex) const override
+    {
+        if (IsHidden(vmItems, nIndex))
+            return L"";
+
+        return ValueColumnBinding::GetText(vmItems, nIndex);
```

Comment:
`ValueColumnBinding` is no longer a base class. This causes a stack overflow.

Reply:
I apparently tested with a test-case that only had hidden values on the source operand, so now I understand how I didn't catch this.